instrumentation/hibernate/hibernate-3.3/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/hibernate/v3_3/SessionTest.java

```diff
@@ -0,0 +1,619 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.hibernate.v3_3;
+
+import static org.junit.jupiter.api.Named.named;
+
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.sdk.testing.assertj.TraceAssert;
+import java.util.function.BiConsumer;
+import java.util.function.Function;
+import java.util.stream.Stream;
+import org.hibernate.LockMode;
+import org.hibernate.Query;
+import org.hibernate.ReplicationMode;
+import org.hibernate.Session;
+import org.hibernate.StatelessSession;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+class SessionTest extends AbstractHibernateTest {
+
+  @ParameterizedTest
+  @MethodSource("provideArguments")
+  void testHibernateAction(Parameter parameter) {
+
+    testing.runWithSpan(
+        "parent",
+        () -> {
+          Session session = sessionFactory.openSession();
+          session.beginTransaction();
+          try {
+            parameter.sessionMethodTest.accept(session, prepopulated.get(0));
+          } catch (RuntimeException e) {
+            // We expected this, we should see the error field set on the span.
```

Comment:
I know this was just brought over as-is, but if the exception really is expected then there should probably be a `fail()` call in the try block.

Reply:
Thank you for the review. So only one of the tests actually threw an exception (`testHibernateFailedReplicate`), so I removed the try/catch from all others, and added an additional event assertion for the exception in the one test where it was applicable. If this wasn't what you had in mind just let me know and I can revisit.

Reply:
Nah that's perfect, thanks! Seems like just leftover copypasta, so I appreciate you cleaning it up.